### PR TITLE
Create script to convert quiz markdown to bqf

### PR DESCRIPTION
**Pull request by LouiseSteward**

### CODE REVIEW

**Comment by LouiseSteward** [comment]
@arthurattwell 

Please take a look at this when you've got some time. I've tested the script on all of the books, hence the changes to a couple of the quizzes - these issues were picked up when I ran the script.

I've also tested the changes to the output script for mac and linux, but had to just eyeball it for the windows script so I'd appreciate you taking an extra careful look at that one.

**Comment by KlaraSkinner** [comment]
@arthurattwell I'll be merging this in with my PR for the Adult HIV update. If you'd like to check and make changes/approve first, let me know, but this has been tested on Windows and Mac.

File: md_to_bqf.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        if '{:.titlepage-title}' in title_src_lines[i]:
+            # the title of the book is in the line above this include
+            book_title = title_src_lines[i - 1].strip('\n')
+
+    title_src.close()
+
+    # get the name of the quiz from the quiz md file
+    quiz_title = src_lines[1].strip('\n').replace('title: ', '')
+
+    # correctly format the dst meta
```
Comment (arthurattwell) [inline md_to_bqf.py]: @LouiseSteward For my info, what does `dst` stand for?

File: md_to_bqf.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+    '''
+    question, answers = question_lines[0], question_lines[1:]
+
+    # format the question by removing the numbering etc.
+    formatted_question = question[re.search(r'\d+\.[\t\s]', question).end():]
+
+    # format the answer by stripping preceding whitespace
+    formatted_answer_list = []
+    for answer in answers:
+        if '+' in answer or '-' in answer:  # exclude empty lines etc.
```
Comment (arthurattwell) [inline md_to_bqf.py]: @LouiseSteward I don't know any Python, so I'm just wondering: do this and the next line only find lines that *start with* `+` or `-`? Or might there be false positives or weird behaviour if a line had, say, a `+` in the text of the answer?

File: md_to_bqf.py (hunk 1 above)
Comment (LouiseSteward) [reply md_to_bqf.py]: @arthurattwell 

dst stands for destination in this instance. It's fairly common in python.

File: md_to_bqf.py (hunk 2 above)
Comment (LouiseSteward) [reply md_to_bqf.py]: @arthurattwell 

Yeah that's a good point - will fix this.

**Pull request merged**